iidm/iidm-modification/src/main/java/com/powsybl/iidm/modification/topology/TopologyModificationUtils.java

```diff
@@ -212,6 +212,7 @@ static void createNodeBreakerSwitches(int node1, int middleNode, int node2, Stri
     static void createNBBreaker(int node1, int node2, String suffix, String prefix, VoltageLevel.NodeBreakerView view, boolean open) {
         view.newSwitch()
                 .setId(prefix + "_BREAKER" + suffix)
+                .setEnsureIdUnicity(true)
```

Comment:
Isn't this quite annoying for some implementations? Indeed from what I remember it builds the index for all network equipments, hence producing a lot of requests for building all the objects of the network.

Reply:
Yes but the issue is if you call any hypotheses more than once (this one in particular but this is valid for all sort of hypotheses), you run the risk of ID collision...

Reply:
Ok, let's forget about performance then. We'll change that when the naming strategy will show up!